### PR TITLE
Add particle textures for electric furnaces

### PR DESCRIPTION
**Pull request by SizzlinSeal01**

### CODE REVIEW

**Comment by SizzlinSeal01** [comment]
this pr does not actually fix the particle issues im trying to identify why it does not want to show the particle textures so this PR just makes the electric furnaces have the particle textures but they dont actually work, making them identical to every other craftory block

**Comment by brettsaunders21** [comment]
Please can you create a new pull request into the 0.2.10 branch instead of master. Thanks

**Comment by TheBusyBiscuit** [comment]
@brettsaunders21 Btw you can change the target branch of the PR yourself when you click "Edit" in the top right corner of the PR.

File: pack/assets/craftory/models/block/powered/furnace/gold_electric_furnace.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	"textures": {
 		"2": "craftory:block/powered/furnace/inner_casing_furnace",
 		"texture": "craftory:block/powered/cores/gold_block"
```
Comment (TheBusyBiscuit) [inline pack/assets/craftory/models/block/powered/furnace/gold_electric_furnace.json:6]: Missing comma 😬

**Review by TheBusyBiscuit — COMMENTED** [review]
@brettsaunders21 You merged ._.

**Pull request merged**